lib/new_relic/agent/log_event_aggregator.rb

```diff
@@ -90,6 +90,42 @@ def record(formatted_message, severity)
         nil
       end
 
+      def record_json(log)
+        return unless enabled?
+
+        severity = log['level'] || 'UNKNOWN'
+
+        if NewRelic::Agent.config[METRICS_ENABLED_KEY]
+          @counter_lock.synchronize do
+            @seen += 1
+            @seen_by_severity[severity] += 1
+          end
+        end
+
+        return if severity_too_low?(severity)
+        # The message key only exists if users manually add log statements
+        return if log.key?(:message) && (log.key?(:message).nil? || llog.key?(:message).empty?)
```

Comment:
```suggestion
        return if log.key?(:message) && (log.key?(:message).nil? || log[:message].empty?)
```

NOTE: 2 changes. `llog` -> `log` and `.key?` -> `[]`

Reply:
New [line](https://github.com/newrelic/newrelic-ruby-agent/pull/2559/files#diff-680b1e6a8547301e66b77f383e273ef141166e5986c9a1180625081dc8e271bcR98)